js/views/composerview.js

```diff
@@ -488,6 +497,8 @@ define(function(require) {
 						$row.append($placeholder);
 					}
 
+					prevUID = item['id'];
```

Comment:
Well, this is a requirement from the contact app and the vcard guidelines.
So I guess it should be? 
https://github.com/owncloud/contacts/blob/master/js/services/contact_service.js#L87